ex1/ex1.js

```diff
@@ -6,3 +6,25 @@
  *     fn(",,,") === 2
  *     fn("hel,lowo,rld") === -1
  */
+
+/////////////////////////////////////////////////////////////////////////////
+// getThirdCommaPosition accepts a string and returns the index of the third comma in that string /  no comma return -1
+function getThirdCommaPosition(str,searcVal) {
+
+	var i, position = -1;
+
+	for (i = 0; i < 3; i++) {
```

Comment:
if you are writing in ES6, I would move the var statement into the for loop and change it to a let, as in `for (let i = 0, ...`.